templates/partials/contacts_content.html

```diff
@@ -92,6 +93,14 @@ <h3 translate>clinic.field.children</h3>
               task-count="child.taskCount"
             />
           </ul>
+          <ul class="action deceased">
```

Comment:
should this just not be a property of sorting? Does it need its own list?

Reply:
It's not a list of the deceased, it's a grey bar that when clicked shows the deceased. I'm using `ul` and `li` so the styling is consistent but it could just as easily be a `div`.